tests/functional/cylc-clean/00-basic.t

```diff
@@ -58,6 +58,15 @@ cmp_ok "${TEST_NAME}.stdout" <<< "${TEST_DIR}/${SYM_NAME}-run/cylc-run/${SUITE_N
 INSTALL_LOG_FILE=$(ls "${TEST_DIR}/${SYM_NAME}-log/cylc-run/${SUITE_NAME}/log/install")
 TEST_NAME="test-dir-tree-pre-clean"
 tree --noreport --charset=ascii "${TEST_DIR}/${SYM_NAME}-"* > "${TEST_NAME}.stdout"
+
+# If rose-cylc plugin is installed add install files to tree.
+export ROSE_FILES=''
+python -c "import cylc.rose" > /dev/null 2>&1 &&
+    export ROSE_FILES="
+                    |-- opt
+                    |   \`-- rose-suite-cylc-install.conf
+                    |-- rose-suite.conf"
+
```

Comment:
I am getting test failures on `tests/functional/cylc-install/03-file-transfer.t` I think similar fix is needed there.

Reply:
Do you have https://github.com/cylc/cylc-rose/pull/20 checked out and pip installed?